app/main/views/signup.py

```diff
@@ -234,6 +234,7 @@ def application_update(id, step=None):
     result = data_api_client.update_application(id, application)
 
     if json:
+        result['application'].pop('links', None)
```

Comment:
use pop because it won't return exception if it's not there?

just seems inconsistent with line 232